test/models/user_test.rb

```diff
@@ -0,0 +1,23 @@
+class UserTest < ActiveSupport::TestCase
```

Comment:
These tests look great! If you don't mind, please add assertion messages so these are easier to debug. I can link you to some documentation about it if you'd like.